docker-compose.prod.yml

```diff
@@ -1,18 +1,32 @@
-version: "3"
+version: "3.7"
 
 services:
+
+  nginx:
+    build: ./nginx
+    image: 410487266669.dkr.ecr.us-west-2.amazonaws.com/spdx/nginx:0.1.0
+    restart: always
+    ports:
+      - 80:80
+    depends_on:
+      - spdx
+      
   spdx:
     container_name: spdx_prod
     build:
       context: ./
       dockerfile: dockerfile.prod
-    ports:
-      - 8000:8000
+    image: 410487266669.dkr.ecr.us-west-2.amazonaws.com/spdx/online-tools:0.1.1
+    expose:
+      - 8000
     volumes:
       - ./container_logs:/spdx/container_logs
+    env_file: spdx-prod.env
     environment:
       - PYTHONDONTWRITEBYTECODE=1
       - SPDX_REDIS_HOST=redis
+      - SPDXParser.LocalLicensesDir=/licenses/current
+      - SPDXParser.OnlyUseLocalLicenses=true
```

Comment:
Where are these variables used?

Reply:
I updated the tools to a new version which supports a cached version of the license to resolve the timeouts for license matching (see https://github.com/spdx/tools/pull/251)  These environment variables provide the location of the cached licenses.